### DISCUSSION TITLE
How to Test Touch Functionality on ED060KD1 Screen

### DISCUSSION THREAD

**Discussion by lixumin-zai** [vroland/epdiy#382] [Q&A]
I am new to this process，😢 I did not find any relevant documents

**Comment by martinberlin** [comment]
Hello @lixumin-zai 
epdiy repository is just about driving the display. Moving this to Discussions since it's really not on topic for the mission of this component. Driving the display is already complex but adding to that that different versions have not the same touch display is already too much and actually the touch needs to be "another component" separated from epdiy.

Please refer to my repository https://github.com/martinberlin/lv_port_esp32-epaper/ where I tried with success to make a driver for "Kindle like" displays
